templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -19,10 +19,10 @@ contents:
       fi
     }
 
-    # download and test etcdctl from upstream release assets
+    # extract etcdctl binary from container image
     dl_etcdctl() {
       local etcdimg="{{.Images.etcdKey}}"
-      local etcdctr=$(podman create "${etcdimg}")
+      local etcdctr=$(podman create "${etcdimg}" --authfile=/var/log/kubelet/config.json)
```

Comment:
There is a typo. The authfile should be /var/lib/kubelet/config.json.

Reply:
updated